jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartFormInputStream.java

```diff
@@ -59,18 +60,17 @@
 public class MultiPartFormInputStream
 {
     private static final Logger LOG = Log.getLogger(MultiPartFormInputStream.class);
-    private static final MultiMap<Part> EMPTY_MAP = new MultiMap<>(Collections.emptyMap());
-    private InputStream _in;
-    private MultipartConfigElement _config;
-    private String _contentType;
-    private MultiMap<Part> _parts;
-    private Throwable _err;
-    private File _tmpDir;
-    private File _contextTmpDir;
-    private boolean _deleteOnExit;
-    private boolean _writeFilesWithFilenames;
-    private boolean _parsed;
-    private int _bufferSize = 16 * 1024;
+    private final MultiMap<Part> _parts = new MultiMap<>();
```

Comment:
Remind me why this is a multimap and not a HashMap<String, Part>? Can multiple parts have the same name?

Reply:
Yes, there can be different parts with the same name so we use MultiMap instead of HashMap.